.github/workflows/cloudbeat-ci.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 env:
   CONTAINER_SUFFIX: ${{ github.run_id }}
-  CI_ELASTIC_AGENT_DOCKER_TAG: "8.7.0-SNAPSHOT"
+  CI_ELASTIC_AGENT_DOCKER_TAG: "8.10.0-SNAPSHOT"
```

Comment:
Did you push this docker image to our ECR?
Anyway, I think we better stop maintaining the docker registry, we can use instead the elastic cloud release
`docker.elastic.co/cloud-release/elastic-agent-cloud:8.10.0-506cd98d-SNAPSHOT`

Reply:
@amirbenun, No, we are using agent snapshots to create a customized version with the latest updates of Cloudbeat. This allows us to test the latest changes in Cloudbeat. So, uploading to ECR cannot be removed.